cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/mapping/CfArgumentsCreator.java

```diff
@@ -0,0 +1,94 @@
+package cloud.foundry.cli.crosscutting.mapping;
+
+import picocli.CommandLine;
+import picocli.CommandLine.ParseResult;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Properties;
+
+import cloud.foundry.cli.crosscutting.logging.Log;
+
+/**
+ * This utility class creates certain options with default values, which were not passed when the command was called.
+ * There are currently default values for <code>api</code>, <code>organization</code> and <code>space</code>.
+ */
+public class CfArgumentsCreator {
+
+    /**
+     * Name of the property file, which contains default values
+     */
+    private static final String CF_CONTROL_PROPERTIES = "cf_control.properties";
+
+    /**
+     * The method determines the commandline arguments.
+     * The following specification applies:
+     * when the command is called and one of the option <code>space</code> or / and <code>api</code>
+     * or / and <code>organization</code> is missing,
+     * the method extends the CommandLine with the corresponding option and
+     * sets a default value for this, which has been configured in a property file.
+     * <p>
+     * For example, it converts the command
+     * "command -s space value -a api value"
```

Comment:
"space value" and "api value" should be written together
```suggestion
     * "command -s spaceValue -a apiValue"
```